quic/state/QuicStreamManager.h

```diff
@@ -73,7 +73,7 @@ class QuicStreamManager {
    * Return the stream state or create it if the state has not yet been created.
    * Note that this is only valid for streams that are currently open.
    */
-  QuicStreamState* getStream(StreamId streamId);
+  QuicStreamState* _Nonnull getStream(StreamId streamId);
```

Comment:
Can you change these to `FOLLY_NONNULL` and `FOLLY_NULLABLE` as is done elsewhere here?

Reply:
Done